rook/README.md

```diff
@@ -77,9 +77,22 @@ $ kubectl create namespace kubestone
 
 ### `iperf`
 ```bash
-$ popper run -f workflows/iperf.yml
+# between server and client-1
+$ popper run -f workflows/iperf.yml --substitution _SERVER=<server> --substitution _CLIENT=<client-1>
+
+# between server and client-2
+$ popper run -f workflows/iperf.yml --substitution _SERVER=<server> --substitution _CLIENT=<client-2>
+
+# between server and client-3
+$ popper run -f workflows/iperf.yml --substitution _SERVER=<server> --substitution _CLIENT=<client-3>
+.
```

Comment:
this is very nice! For completeness, this could show an example of what values are expected for `_CLIENT and `_SERVER`, are those supposed to be hostname/IP addresses?